bigtable/api/QuickStart/QuickStart.cs

```diff
@@ -0,0 +1,56 @@
+﻿// [START bigtable_quickstart]
+using System;
+// Imports the Google Cloud client library
+using Google.Cloud.Bigtable.V2;
+
+namespace GoogleCloudSamples.Bigtable
+{
+    class QuickStart
+    {
+        static int Main(string[] args)
+        {
+            // Your Google Cloud Platform project ID
+            const string projectId = "YOUR-PROJECT-ID";
+            // The name of the Cloud Bigtable instance
+            const string instanceId = "YOUR-INSTANCE-ID";
+            // [END bigtable_quickstart]
+            if (projectId == "YOUR-PROJECT" + "-ID")
+            {
+                Console.WriteLine("Edit QuickStart.cs and replace YOUR-PROJECT-ID with your project id.");
+                return -1;
+            }
+            if (instanceId == "YOUR-INSTANCE" + "-ID")
+            {
+                Console.WriteLine("Edit QuickStart.cs and replace YOUR-INSTANCE-ID with your instance id.");
+                return -1;
+            }
+            // [START bigtable_quickstart]
```

Comment:
Please use only one pair of the region tag in the same file. Remove the nested START / END bigtable_quickstart tags

Reply:
I got it from this pubsub example
https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/c391b70b5f96489815ed173c1b619bcf317539c5/pubsub/api/QuickStart/QuickStart.cs#L32-L38

I understood that when documentation will be generated the lines between END and START will be omitted.
Should the nested tags be removed?

Reply:
Aah, sorry I was wrong here : did n't realize it would render right. You're good then.